src/cli/probe.ts

```diff
@@ -0,0 +1,64 @@
+import { inputData, outputData } from '../lib/io';
+import { invokeUserFn, loadUserFn } from './lib/user-fn';
+import { verifyInputArray } from '../lib/verify';
+import {
+  standardCmdInputs,
+  standardCmdOutputs,
+  standardInputFileHelp,
+  standardOutputFileHelp,
+} from './lib/help';
+
+export async function main(argv: string[]): Promise<void> {
+  const { fn, details } = loadUserFn(argv, `record => transform(record)`);
+
+  const data = await inputData(argv);
+  verifyInputArray(data, 'map');
+
+  data.map((record: any) => {
+    const result = invokeUserFn(() => fn(record), details);
+    console.log(result);
+    return record;
```

Comment:
Once the `map` above is changed to `forEach` this return statement can be removed.